paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/utils/FileUtils.kt

```diff
@@ -19,9 +19,10 @@ import org.gradle.api.file.ConfigurableFileCollection
 import org.gradle.api.file.Directory
 import java.io.File
 
-fun ConfigurableFileCollection.joinFiles(directory: Directory) = files.joinToString(",") { file ->
-  directory.relativize(file)
-}
+fun ConfigurableFileCollection.joinFiles(directory: Directory, reversed: Boolean = false) =
+  files.run { if (reversed) reversed() else this }.joinToString(",") { file ->
+    directory.relativize(file)
+  }
```

Comment:
Adding a boolean flag to functions is [usually a code smell](https://blog.jetbrains.com/idea/2017/09/code-smells-multi-responsibility-methods/) (long read, but awesome series!).

What do you think about
```suggestion
fun Iterable<File>.joinFiles(directory: Directory): String =
  joinToString(",") { file -> directory.relativize(file) }
```
which would change the call-site to
```kotlin
        it.write(projectResourceDirs.reversed().joinFiles(projectDirectory))
        it.newLine()
        it.write(moduleResourceDirs.joinFiles(projectDirectory))
        it.newLine()
        it.write(aarExplodedDirs.joinFiles(gradleUserHomeDirectory))
```
Notice how the other `joinFiles` calls stay the same!